packages/integrations/sagas/picoSaga.js

```diff
@@ -108,6 +108,6 @@ function trackUpdatePicoSignal(signal) {
   window.dataLayer = window.dataLayer ?? [];
   window.dataLayer.push({
     event: 'irving.updatePicoSignal',
-    picoSignal: signal,
+    picoSignal: signal.payload,
```

Comment:
Maybe add a check for this property before trying to access it to prevent potential JS error.